src/lib/stitching/causality/stitching.ts

```diff
@@ -24,37 +28,15 @@ const resolveLotCentsFieldToMoney = (centsField) => {
 export const causalityStitchingEnvironment = ({
   causalitySchema,
   localSchema,
+  version,
 }: {
   causalitySchema: GraphQLSchema & { transforms: any }
   localSchema: GraphQLSchema
+  version: number
 }) => {
   return {
     extensionSchema: gql`
-      type Lot {
-        internalID: String
-        lot: AuctionsLotState!
-        saleArtwork: SaleArtwork
-      }
-
-      # A connection to a list of items.
-      type LotConnection {
-        # A list of edges.
-        edges: [LotEdge]
-        pageCursors: PageCursors!
-
-        # Information to aid in pagination.
-        pageInfo: PageInfo!
-        totalCount: Int
-      }
-
-      # An edge in a connection.
-      type LotEdge {
-        # A cursor for use in pagination
-        cursor: String!
-
-        # The item at the end of the edge
-        node: Lot
-      }
+      ${version === 2 ? stitchedCausalityLotExtensionSchema : ""}
```

Comment:
simplified addition to our extension schema.

Reply:
This comment still stands: https://github.com/artsy/metaphysics/pull/2928#discussion_r564747657

Reply:
Yea @damassi , by default we still apply the stitching and transformations to V1 and V2. You need to manually do this to avoid updating V1 schema. We do this in several places throughout our stitching infra, and prob will be happening more as future code is likely to refer or extend types which themselves don't even exist in V1.

Reply:
👍 I'm just a bit unsure why, for example, convection stitching didn't include this. We're only stitching stuff into v2, there was no stitching in v1, same with Gravity, but I could be wrong about this.

Reply:
It's definitely stitched in both places. Check out the [V1 Schema](https://github.com/artsy/metaphysics/blob/54d603e12c745ff67116c65107d307f1ac06f3d8/_schema.graphql), lots of mentions of consignment stuff